app/models/site.rb

```diff
@@ -16,4 +16,8 @@ class Site < Base
     ['Scotland', :scotland],
     ['No Region', :no_region],
   ].freeze
+
+  def full_address
+    [address1, address2, address3, address4, postcode].select(&:present?).join("<br> ").html_safe
```

Comment:
Will this put a line break after each line?

It currently shows like this on Find:
![Screen Shot 2019-05-29 at 11 30 30](https://user-images.githubusercontent.com/319055/58550235-3b215b80-8205-11e9-92d4-94dc9603ebfa.png)

Reply:
Yep, my bad. Remove the line break and replaced with a comma